config/gregtech/worldgen/overworld/basalt_sphere.json

```diff
@@ -1,28 +0,0 @@
-{
```

Comment:
We should create a method to still obtain these blocks since they no longer spawn. Their dusts have products, and might be nice for building.